InteractiveHtmlBom/web/render.js

```diff
@@ -518,24 +551,20 @@ function handlePointerMove(e, layerdict) {
 
   var thisPtr = layerdict.pointerStates[e.pointerId];
 
-  if (Object.keys(layerdict.pointerStates).length == 1) {
-    // This is a simple drag
-    var dx = e.offsetX - thisPtr.lastX;
-    var dy = e.offsetY - thisPtr.lastY;
+  var dx = e.offsetX - thisPtr.lastX;
+  var dy = e.offsetY - thisPtr.lastY;
 
-    // If this number is low on pointer up, we count the action as a click
-    thisPtr.distanceTravelled += Math.abs(dx);
-    thisPtr.distanceTravelled += Math.abs(dy);
+  // If this number is low on pointer up, we count the action as a click
+  thisPtr.distanceTravelled += Math.abs(dx);
+  thisPtr.distanceTravelled += Math.abs(dy);
```

Comment:
Combine these lines with `+= Math.abs(dx) + Math.abs(dy)` for brevity

Reply:
`dx`/`dy` are used later to figure out how much to pan by. Should I recalculate them later?

Reply:
No, keep the variables, just increment distanceTravelled in one statement instead of two.

Reply:
Oh whoops, understood :)